input/elasticapm/internal/modeldecoder/v2/span_test.go

```diff
@@ -174,9 +174,16 @@ func TestDecodeMapToSpanModel(t *testing.T) {
 		assert.Equal(t, "failure", out.Event.Outcome)
 		// derive from other fields - unknown
 		input.Outcome.Reset()
+		input.OTel.Reset()
```

Comment:
The change was only made to OTel input for now, so I needed to reset the Otel attributes here so we don't get confused why it affects also `elasticapm` input